recipes-support/onboard/files/0002-onboard-onhover-seg-fault-fix.patch

```diff
@@ -0,0 +1,52 @@
+From: 1be95325d320122efd5dedf7437839cfcca01f7a
+From: https://github.com/void-linux/void-packages/commit/1be95325d320122efd5dedf7437839cfcca01f7a
+Date: Mon, 23 Sep 2024
+Subject: [PATCH] onboard: fix segfault when hovering over the keyboard
+
+Currently, if the mouse pointer is hovered over the Onboard keyboard, the application crashes with a segmentation fault.
+This is because the new_device_event function is not acquiring the GIL before creating a new object. This patch fixes
+the issue by acquiring the GIL before creating a new object. It also acquires the GIL before queueing the event.
+This patch is taken from the fix provided in the following link:
+https://github.com/void-linux/void-packages/commit/1be95325d320122efd5dedf7437839cfcca01f7a
+
+Signed-off by: Pratheeksha S N <pratheeksha.s.n@ni.com>
+
+Upstream-Status: Not Submitted [This is being added to the meta-nilrt layer for now. If it is accepted upstream, it will be removed from here.]
```

Comment:
Oh I forgot about this patch.. if you're planning to send this upstream along with https://github.com/ni/meta-openembedded/pull/67, you should ideally put this file in that PR so that other NI folk can review the final to-be-sent-upstream changes.

Reply:
Have moved the patch to meta-openembedded now.

Reply:
I see that the file is still here.. I suppose you're planning to remove it here once upstream accepts meta-oe changes? In that case let's keep this issue open so we don't accidentally merge this PR. (Also, the file is not usable in the PR anyway as it isn't included `SRC_URI`.

Reply:
I have deleted the file now.